lib/spack/spack/config.py

```diff
@@ -95,6 +95,7 @@
         "debug": False,
         "connect_timeout": 10,
         "verify_ssl": True,
+        "git_terminal_prompt_enable": False,
```

Comment:
I don't think we need a config option for this -- we can just hard-disable it in the git fetch strategy, unless I'm missing something.